paddle/fluid/pybind/imperative.cc

```diff
@@ -628,6 +632,28 @@ void BindImperative(py::module *m_ptr) {
                return out;
              }
            })
+      .def("_inplace_version",
+           [](imperative::VarBase &self) -> uint32_t {
+             const auto &var = self.MutableVar();
+             PADDLE_ENFORCE_EQ(
+                 var->IsInitialized(), true,
+                 platform::errors::InvalidArgument(
+                     "Tensor of %s is Empty, please check if it has no data.",
+                     self.Name()));
+
+             return var->InplaceVersionCounter().CurrentVersion();
+           })
+      .def("bump_inplace_version",
```

Comment:
Does this API need to be public, maybe `_bump_implace_version` is better?

Reply:
No, thanks for your suggestions.  Done